src/Sylius/Bundle/SearchBundle/Finder/OrmFinder.php

```diff
@@ -268,7 +268,9 @@ public function buildFacet($idsFromOtherFacets, $facets, $givenFacetName, $resul
                             }
                         }
                     }
-                    asort($calculatedFacets[$name]);
+                    if (isset($calculatedFacets[$name]) && !is_null($calculatedFacets[$name])) {
```

Comment:
This `is_null()` is useless as `isset()` don't allow `null` anyway... http://3v4l.org/5CPoY